docs/reference/security-hardening/webhook-mtls.md

```diff
@@ -0,0 +1,188 @@
+
+The built-in NetworkPolicy resource in Kubernetes can't block or admit traffic from the cluster hosts, and the `kube-apiserver` process is always running on the host network.
+Therefore, you must use the advanced network policy resources from the CNI in use. Examples for Calico and Cilium follow. Consult the documentation for your CNI for more details.
+
+### Calico
+
+Use the NetworkPolicy resource in the `crd.projectcalico.org/v1` API group, define a network policy like the following one:
+
+```yaml
+apiVersion: crd.projectcalico.org/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-k8s-and-audit-scanner
+  namespace: kubewarden # namespace where the kubewarden stack is deployed
+spec:
+  selector: 'app.kubernetes.io/name == "kubewarden-controller" || has(kubewarden/policy-server)'
+  types:
+    - Ingress
+  ingress:
+    # this allows the Kubernetes API server to reach the kubewarden controller and
+    # all the policy server instances
+    - action: Allow
+      protocol: TCP
+      source:
+        nets:
+        # CIDR of the control plane host. May list more than 1 if the hosts are in different subnets.
+        - 192.168.42.0/24
+      destination:
+        selector:
+          'app.kubernetes.io/name == "kubewarden-controller" || has(kubewarden/policy-server)'
+    # this allows all the workloads defined inside of the kubewarden namespace
+    # (including audit-scanner) to reach the kubewarden controller and all the
+    # policy server instances
+    - action: Allow
+      protocol: TCP
+      source:
+        # namespace where the kubewarden stack is deployed
+        namespaceSelector: kubewarden
+      destination:
+        selector:
+          'app.kubernetes.io/name == "kubewarden-controller" || has(kubewarden/policy-server)'
+```
+
+### Cilium
+
+Use the CiliumNetworkPolicy resource in the `cilium.io/v2` API group, define a network policy like the following one:
+
+```yaml
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: allow-k8s-and-audit-scanner
+  namespace: kubewarden # namespace where the kubewarden stack is deployed
+spec:
+  endpointSelector:
+    matchExpressions:
+      - key: app.kubernetes.io/name
+        operator: In
+        values:
+          - kubewarden-controller
+      - key: kubewarden/policy-server
+        operator: Exists
+  ingress:
+    # this allows the Kubernetes API server to reach the kubewarden controller and
+    # all the policy server instances
+    - fromEntities:
+      - host
+      - remote-node
+    # this allows all the workloads defined inside of the kubewarden namespace
+    # (including audit-scanner) to reach the kubewarden controller and all the
+    # policy server instances
+    - fromEndpoints:
+        - matchLabels:
+            # namespace where the kubewarden stack is deployed
+            k8s:io.kubernetes.pod.namespace: kubewarden
+```
+
+## Require the Kubernetes API Server to Authenticate to the Webhook
```

Comment:
As this section is pretty similar to the on in the 'howto' page, shouldn't we mention that page in case the user want to see a full example on how to enable it?

Reply:
Good point, I've added a `tip` section that contains a reference to the howto